.vsts-ci/linux.yml

```diff
@@ -80,24 +80,17 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      Import-Module ./tools/ci.psm1
+      Invoke-Bootstrap-Stage
     displayName: Bootstrap
     condition: succeeded()
 
   - powershell: |
       $ErrorActionPreference = 'continue'
```

Comment:
Does it work for module below?

Reply:
It should

Reply:
I moved it into the PSM1 and out of the YAMLs